packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,28 @@
+{
+  "netapp_volumes_snapshots": "Snapshots",
+  "netapp_volumes_snapshots_breadcrumb": "Snapshots",
+  "netapp_volumes_snapshots_total_label": "Nombre de snapshots",
+  "netapp_volumes_snapshots_total": "{{ total }} / {{ max }} (maximum)",
+  "netapp_volumes_snapshots_actions": "Actions",
+  "netapp_volumes_snapshots_actions_create": "Créer un snapshot",
+  "netapp_volumes_snapshots_actions_create_label": "Snapshot: Créer un snapshot",
+  "netapp_volumes_snapshots_actions_edit": "Éditer",
+  "netapp_volumes_snapshots_actions_delete": "Supprimer",
+  "netapp_volumes_snapshots_id": "ID",
+  "netapp_volumes_snapshots_name": "Nom",
+  "netapp_volumes_snapshots_description": "Description",
+  "netapp_volumes_snapshots_created_at": "Date de création",
+  "netapp_volumes_snapshots_type": "Type",
+  "netapp_volumes_snapshots_type_manual": "Manuelle",
+  "netapp_volumes_snapshots_status": "Statut",
+  "netapp_volumes_snapshots_status_available": "Disponible",
+  "netapp_volumes_snapshots_status_error": "Erreur",
+  "netapp_volumes_snapshots_status_creating": "Création en cours",
+  "netapp_volumes_snapshots_status_deleting": "Suppression en cours",
+  "netapp_volumes_snapshots_status_error_deleting": "Suppression en erreur",
+  "netapp_volumes_snapshots_status_manage_starting": "Disponible",
+  "netapp_volumes_snapshots_status_manage_error": "Disponible",
+  "netapp_volumes_snapshots_status_unmanage_starting": "Disponible",
+  "netapp_volumes_snapshots_status_unmanage_error": "Disponible",
```

Comment:
Same content ?

Reply:
This was the content provided for all these status